f5/bigip/tm/cm/trust.py

```diff
@@ -31,13 +31,17 @@ class Add_To_Trust(UnnamedResourceMixin, ExclusiveAttributesMixin,
 
     def __init__(self, cm):
         super(Add_To_Trust, self).__init__(cm)
+        base_uri = type(self).__name__.replace('_', '-').lower()
+        self._meta_data['uri'] =\
+            self._meta_data['container']._meta_data['uri'] + base_uri + '/'
```

Comment:
we can address this later as this is not the only place you have this logic in the code